src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:22908/organizations/org2/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:22908/project/issues?id=sample&pullRequest=feature3&issues=AWG_v8rj9b60XQ4axc2U&open=AWG_v8rj9b60XQ4axc2U)

[ISSUE_KEY]: AWG_v8rj9b60XQ4axc2U